modules/deployments-blue-green.adoc

```diff
@@ -49,7 +49,7 @@ $ oc new-app openshift/deployment-example:v2 --name=example-green
 $ oc expose svc/example-blue --name=bluegreen-example
 ----
 
-. Browse to the application at `example-blue.<project>.<router_domain>` to verify you see the `v1` image.
+. Browse to the application at `bluegreen-example.<project>.<router_domain>` to verify you see the `v1` image.
```

Comment:
Better to change as "bluegreen-example-<project>.<router_domain>", because the route is like this: 
bluegreen-example-zhouy.apps.yinzhou0624.qe.devcluster.openshift.com